person.rb

```diff
@@ -18,4 +21,8 @@ def can_use_services?
   def of_age?
     @age.to_i >= 18
   end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] @kessie2862 Great work adding the correct_name method however it should be public to use it when displaying the person's name.

Reply:
- [ ] [optional] It would be nice if you added a test.rb file, which the user can run to see the decoration in action, you can use the sample code provided 
![image](https://github.com/kessie2862/school-library/assets/61408860/8deccb1f-e2e3-4943-9053-6a3623c66a2d)